### PR TITLE
fix: shell scripts

### PR DESCRIPTION
**Pull request by martabal**
Fix shell scripts which don't respect [shellcheck](https://github.com/koalaman/shellcheck) syntax.

Files found with this command : `find ./ -type f -name "*.sh" -not \( -path "**/open-api/**" -o -path "**/openapi/**" -o -path "**/node_modules/**" \) -exec shellcheck --format=gcc {} \;`

The `pump-version.sh` script bump the web version too.

### CODE REVIEW

**Comment by martabal** [comment]
I don't know if it's worth adding a CI job for this

File: install.sh [hunk 1]
```diff
@@ -1,15 +1,13 @@
+#!/bin/bash
```
Comment (mertalev) [inline install.sh:1]: Change this to `#!/usr/bin/env bash` (also for the others, but keep `sh` as `sh`)

**Review by mertalev — APPROVED** [review]
Nice! I think a check would be fine if you'd like to add one

**Review by danieldietzler — APPROVED** [review]
So it isn't considered good practice to assign a value to an env variable directly? (without an intermediate variable)

**Comment by martabal** [comment]
> So it isn't considered good practice to assign a value to an env variable directly? (without an intermediate variable)

Yes, if you use `export` https://www.shellcheck.net/wiki/SC2155

**Comment by mertalev** [comment]
> In the original code, the return value of mycmd is ignored, and export will instead always return true. This may prevent conditionals, set -e and traps from working correctly.

Oh wow, I didn't know it ignored return values.

**Review by jrasm91 — APPROVED** [review]

**Pull request merged**